packages/react/src/components/TableOfContents/TableOfContents.js

```diff
@@ -212,9 +213,7 @@ const TableOfContents = ({
             <div className={`${prefix}--tableofcontents__desktop__children`}>
               {headingContent}
 
-              {menuRule ? (
-                <hr className={`${prefix}--tableofcontents__desktop__rule`} />
-              ) : null}
+              {menuRule ? <HorizontalRule /> : null}
```

Comment:
@ariellalgilmore small thing, but this can be rewritten as:

```suggestion
              {menuRule && (
                <HorizontalRule />
              )}
```

(So can the containing `headingContent` conditional render)